Assets/VRM/UniJSON/Scripts/JsonSchemaValidator/JsonStringValidator.cs

```diff
@@ -111,10 +111,6 @@ public JsonSchemaValidationException Validate<T>(JsonSchemaValidationContext c,
             }
 
             var value = o as string;
-            if (value.All(x => Char.IsWhiteSpace(x)))
```

Comment:
json schema allows empty string

Reply:
null check ?

Reply:
Thanks, I'll add null checks for all validators.